salesagility/SuiteCRM
# Fix/5639 subpanel compose email

Fixes a few bug related to the compose email button in a records' subpanel.

## Description



1. Fixes an issue with compose buttons not being displayed at all due to the opt in span throwing off a jquery find.
2. Correctly aligns the compose button with the other buttons in the subpanel dropdown.
3. Fixes an issue with Cases compose modal not showing the related record and not picking up any of the information from it (#5639 ).

## Motivation and Context

Fix bugs

## How To Test This

1. Enter a contact/account record detail view
2. See the subpanel compose email button shows
3. See the correct email is displayed as the button caption
4. See the buttton is correctly aligned
5. Enter a case record
6. Press compose email button in subpanel
7. See the compose modal is correctly filled with data

## Types of changes


### Final checklist
